cg/cli/workflow/commands.py

```diff
@@ -0,0 +1,188 @@
+import logging
+import shutil
+from pathlib import Path
+import datetime as dt
+
+import click
+from dateutil.parser import parse as parse_date
+
+from cg.constants import EXIT_SUCCESS, EXIT_FAIL
+from cg.exc import FlowcellsNeededError
+from cg.meta.workflow.analysis import AnalysisAPI
+from cg.meta.workflow.balsamic import BalsamicAnalysisAPI
+from cg.meta.workflow.mip_dna import MipDNAAnalysisAPI
+
+OPTION_DRY = click.option(
+    "-d", "--dry-run", "dry_run", help="Print command to console without executing", is_flag=True
```

Comment:
Looks like this can be reused, think I saw the exact same option defined earlier
```suggestion
    "-d", "--dry-run", help="Print command to console without executing", is_flag=True
```